twilight-mention/src/fmt.rs

```diff
@@ -142,7 +142,8 @@ impl Mention<Id<EmojiMarker>> for Id<EmojiMarker> {
 /// Mention an emoji. This will format as `<:emoji:ID>`.
 impl Mention<Id<EmojiMarker>> for Emoji {
     fn mention(&self) -> MentionFormat<Id<EmojiMarker>> {
-        MentionFormat(self.id)
+        assert!(self.id.is_some(), "Emoji has no id");
+        MentionFormat(self.id.unwrap())
```

Comment:
This should return a result instead of panicking, or at least, it should be documented that it panics when `Emoji.id` is `None`